app/scripts/directives/oscFileInput.js

```diff
@@ -24,7 +23,7 @@ angular.module('openshiftConsole')
         var dropMessageSelector = "#" + scope.dropMessageID,
             highlightDropZone = false,
             showDropZone = false,
-            inputFileField = element.find('input[type=file]')[0];
+            inputFileField = element.find('input[type=file]');
```

Comment:
nit: we could do `inputFileField = element.find('input[type=file]')[0];` or `inputFileField = _.first(element.find())` here & not have to repeat the `[0]` a few places below.